REQUIRE

```diff
@@ -1,6 +1,8 @@
-julia 0.6
+julia 0.7
 JuMP 0.17.1
 Ipopt
 Combinatorics
 RecipesBase
 LaTeXStrings
+MathProgBase
+LinearAlgebra
```

Comment:
LinearAlgebra does not need to be included here

Reply:
yes. I just removed it and added Project.toml. Tests should work fine now